README.md

```diff
@@ -162,6 +162,18 @@ pip install "finetuner[full]"
 > ⚠️ Starting with version 0.5.0, Finetuner computing is performed on Jina AI Cloud. The last local version is `0.4.1`. 
 > This version is still available for installation via `pip`. See [Finetuner git tags and releases](https://github.com/jina-ai/finetuner/releases).
 
+<!-- start finetuner-articles -->
+## Articles about Finetuner
+
+There are several articles about Finetuner and tutorials which using our framework:
```

Comment:
i would rephrase it a bit as something like: we have created several articles, all blog posts below are our "internal" blog post

Reply:
yes you are right, first I thought we can add some external as well but I did not found good ones to add.